prow/github/client.go

```diff
@@ -199,6 +199,20 @@ type MilestoneClient interface {
 type RerunClient interface {
 	TeamHasMember(teamID int, memberLogin string) (bool, error)
 	GetTeamBySlug(slug string, org string) (*Team, error)
+	AddLabel(org, repo string, number int, label string) error
+	BotName() (string, error)
+	IsCollaborator(org, repo, user string) (bool, error)
+	IsMember(org, user string) (bool, error)
+	GetPullRequest(org, repo string, number int) (*PullRequest, error)
+	GetRef(org, repo, ref string) (string, error)
+	CreateComment(owner, repo string, number int, comment string) error
+	ListIssueComments(owner, repo string, issue int) ([]IssueComment, error)
+	CreateStatus(owner, repo, ref string, status Status) error
+	GetCombinedStatus(org, repo, ref string) (*CombinedStatus, error)
+	GetPullRequestChanges(org, repo string, number int) ([]PullRequestChange, error)
+	RemoveLabel(org, repo string, number int, label string) error
+	DeleteStaleComments(org, repo string, number int, comments []IssueComment, isStale func(IssueComment) bool) error
+	GetIssueLabels(org, repo string, number int) ([]Label, error)
```

Comment:
You should only need to add `IsCollaborator`, `IsMember`, and `GetIssueLabels`.  Currently you need all these because `trigger.githubClient` provides more methods than `trigger.TrustedPullRequest()` actually needs.